### ISSUE TITLE
Do we need a Patcher CLI to help maintainers?

### ISSUE THREAD

**Issue by ina-stoyanova** [gruntwork-io/gruntwork-io.github.io#533]
- [x] Think through a bit more if we need a patcher CLI to help maintainers test their patches.
- [x] Do we need this by Friday? If no, change the milestone to `patcher-v2`

**Comment by ina-stoyanova** [comment]
This is also related to #541

**Comment by ina-stoyanova** [comment]
I'll move this to patcher-v2. It just doesn't feel that much of a initial launch feature.

**Comment by ina-stoyanova** [comment]
Check out the [DependaBot page in the web archive](https://web.archive.org/web/20190323092226/https://dependabot.com/) for inspiration on how they tell their story.

**Comment by ina-stoyanova** [comment]
I believe this is being answered with the decisions being made here: https://github.com/gruntwork-io/gruntwork-io.github.io/pull/557 and here with the gifs: https://github.com/gruntwork-io/gruntwork-io.github.io/pull/584

TLDR: It seems like we've all ended up going the CLI way, and the `.patch` files. 

Is there any more decisions to be made here?

**Comment by brikis98** [comment]
Agreed, all decisions have been made, closing.
